pkgs/development/guile-modules/guile-git/default.nix

```diff
@@ -34,9 +29,16 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = [
     libgit2 scheme-bytestructures
   ];
+  doCheck = true;
+  makeFlags = [ "GUILE_AUTO_COMPILE=0" ];
```

Comment:
I couldn't find why we want to set this or what this does.

Also next time it would probably be easier to combine the 3 PRs.

Reply:
This is addressed from your review at #189421.

(Yeah, I should've put this into a single PR to make reviewing easier since the changes are very similar anyways.)